examples/background-color.html

```diff
@@ -64,7 +64,7 @@ <h2>With background-color</h2>
             <template>
                 <model-viewer src="assets/shishkebab.glb"
                     alt="A 3D model of a shishkebab"
-                    controls auto-rotate background-color="#ccc"></model-viewer>
+                    controls auto-rotate background-color="#ddd"></model-viewer>
```

Comment:
The bottom of the plate blended into the background, so I took a stab at a new color. Happy to tweak further.

Before:
![camo-shiskebab](https://user-images.githubusercontent.com/562883/51946938-0678d100-23d8-11e9-8d4f-88248934157a.png)

After:
![less-camo-shiskebab](https://user-images.githubusercontent.com/562883/51946942-0a0c5800-23d8-11e9-88be-ee7f432d588a.png)